cpp/src/parquet/bloom_filter.h

```diff
@@ -36,7 +36,7 @@ class PARQUET_EXPORT BloomFilter {
  public:
   // Maximum Bloom filter size, it sets to HDFS default block size 128MB
   // This value will be reconsidered when implementing Bloom filter producer.
-  static constexpr uint32_t kMaximumBloomFilterBytes = 128 * 1024 * 1024;
+  static constexpr uint64_t kMaximumBloomFilterBytes = 128 * 1024 * 1024;
```

Comment:
Should the return type of `OptimalNumOfBits` be changed to uint64_t as well?

Reply:
Currently just keep it uint32_t is ok, since it could support 4G. For `Bits`, its 1G Bytes, and it's 8 times greater than `kMaximumBloomFilterBytes`.